### DISCUSSION TITLE
Welcome to JokesOnYou Discussions!

### DISCUSSION THREAD

**Discussion by HellHunterMax** [Code2Gether-Discord/JokesOnYou#23] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by purnaanrup** [comment]
can you update your discord link

**Reply by HellHunterMax** [reply]
We well get a new link ASAP and update you here.

**Reply by HellHunterMax** [reply]
Its Updated thanks for your comment!
